packages/internal/src/is-passable.js

```diff
@@ -0,0 +1,23 @@
+import { passStyleOf } from '@endo/pass-style';
+
+/**
+ * Is `specimen` Passable? This returns true iff `passStyleOf(specimen)`
+ * returns a string. This returns `false` iff `passStyleOf(specimen)` throws.
+ * Under no normal circumstance should `isPassable(specimen)` throw.
+ *
+ * TODO implement an isPassable that does not rely on try/catch, and
+ * move it to @endo/pass-style.
+ * This implementation is just a standin until then
```

Comment:
I thought that `@agoric/internal` was a decent temporary place for this until it is moved to `@endo/pass-style`. But it turns out that it creates a dependency cycle (thanks CI) because `@agoric/internal` currently depends on `@agoric/zone`. 

IIUC, this temporarily violates the intention that `@agoric/internal` is at the bottom of the agoric-sdk dependency graph. But seeing as we expect most of `@agoric/zone` to migrate to `@endo/zone`, perhaps we should not fix `@agoric/internal` to stop depending on Zones?

Reply:
@turadg just so you're aware of the dependency violation. Note that I suggest above why IMO it does not need to be fixed.

Reply:
Your understanding is correct and I endorse your conclusion. :)